packages/typespec-client-generator-core/test/types.test.ts

```diff
@@ -2462,6 +2462,25 @@ describe("typespec-client-generator-core: types", () => {
       strictEqual(kindProperty.type, kind);
     });
 
+    it("response header with enum value", async () => {
+      await runner.compileWithBuiltInService(`
+      model RepeatableResponse {
+        @visibility("read")
+        @header("Repeatability-Result")
+        repeatabilityResult?: "accepted" | "rejected";
+      }
+      op foo(): RepeatableResponse;
+      `);
+      const sdkPackage = runner.context.experimental_sdkPackage;
+      strictEqual(sdkPackage.models.length, 0);
+      strictEqual(sdkPackage.enums.length, 1);
+      strictEqual(sdkPackage.enums[0].name, "FooResponse");
```

Comment:
@tadelesh what are your thoughts on this generated name?

Reply:
i think we need to add header name.

Reply:
committed the code.